AK/Math.h

```diff
@@ -660,6 +660,15 @@ constexpr T pow(T x, T y)
     return exp2<T>(y * log2<T>(x));
 }
 
+template<FloatingPoint T>
+constexpr T fma(T x, T y, T z)
+{
+    CONSTEXPR_STATE(fma, x, y, z);
+
+    // FIXME: Probably should get an optimized implementation.
+    return (x * y) + z;
+}
+
```

Comment:
I think it suffices to be implemented in Libm,
The compilers usually are smart enough to replace these with the corresponding instruction, if available

Reply:
Will do.

On that topic, is there a particular reason for the split between `AK/Math.h` and the rest of `LibM`? What dictates which goes where?

Reply:
Ak math was intended to get the things that are constexpr or are better inlined than called,
I think some of this is resolved by nomath-errno
It also allows us to write more complex code in a generic way or force the compiler to inline it in a way we want
An additional thing is that we dont always need to do c99 here, but can just add the helpers we need like to_rounded which resolves more cases than lround and is easily inlined